gufe/components/explicitmoleculecomponent.py

```diff
@@ -82,15 +83,18 @@ def _check_partial_charges(mol: RDKitMol) -> None:
                 raise ValueError(errmsg)
 
     if np.all(np.isclose(p_chgs, 0.0)):
-        wmsg = f"Partial charges provided all equal to " "zero. These may be ignored by some Protocols."
+        wmsg = "Partial charges provided all equal to zero. These may be ignored by some Protocols."
         warnings.warn(wmsg)
     else:
-        wmsg = (
+        message = (
```

Comment:
Should we add something that can be used to identify the SMC this is raised for maybe just adding the `ofe-name` prop to the message?

Reply:
we don't require SMCs to have names, is there a better identifier?

Reply:
Good point how about `ofe-name` if present else `self.smiles` then we should always be able to workout which molecule the message refers to.

Reply:
From offline discussion,  we decided that adding `self.smiles` would be helpful, but requires a bit of refactoring that we'd like to not include in this PR. 

https://github.com/OpenFreeEnergy/gufe/issues/473